featurebyte/schema/info.py

```diff
@@ -302,29 +302,26 @@ def from_paginated_data(cls, paginated_data: dict[str, Any]) -> FeatureListBrief
         return FeatureListBriefInfoList(__root__=feature_list_project.project(paginated_data))
 
 
-class FeatureListInfo(NamespaceInfo):
+class FeatureListNamespaceInfo(NamespaceInfo):
     """
-    FeatureList info schema
+    FeatureListNamespace info schema
     """
 
     dtype_distribution: List[FeatureTypeFeatureCount]
+    default_feature_list_id: PydanticObjectId
     status: FeatureListStatus
     feature_count: int
-    version: VersionComparison
-    production_ready_fraction: ProductionReadyFractionComparison
-    versions_info: Optional[FeatureListBriefInfoList]
-    deployed: bool
 
 
-class FeatureListNamespaceInfo(NamespaceInfo):
+class FeatureListInfo(FeatureListNamespaceInfo):
     """
-    FeatureListNamespace info schema
+    FeatureList info schema
     """
 
-    dtype_distribution: List[FeatureTypeFeatureCount]
-    default_feature_list_id: PydanticObjectId
-    status: FeatureListStatus
-    feature_count: int
+    version: VersionComparison
+    production_ready_fraction: ProductionReadyFractionComparison
+    versions_info: Optional[FeatureListBriefInfoList]
+    deployed: bool
```

Comment:
The main changes here are
* to make `FeatureListInfo` inherits `FeatureListNamespaceInfo` instead of `NamespaceInfo`
* to remove duplicated shared attributes after the inheritance update